shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -126,8 +137,27 @@ void PlatformViewChannel::HandleMethodCall(
         it->second->resize(width, height);
         result->NotImplemented();
       } else if (method == "touch") {
-        LoggerD("PlatformViewChannel touch");
-        result->NotImplemented();
+        int type, button;
+        double x, y, dx, dy;
+
+        flutter::EncodableList event = extractListFromMap(arguments, "event");
+        if (event.size() != 6) {
+          result->Error("Invalid Arguments");
+          return;
+        }
+        type = std::get<int>(event[0]);
+        button = std::get<int>(event[1]);
+        x = std::get<double>(event[2]);
+        y = std::get<double>(event[3]);
+        dx = std::get<double>(event[4]);
+        dy = std::get<double>(event[5]);
+
+        // LoggerD("Engine::Touch type[%s],btn[%d],x[%f],y[%f],dx[%f],dy[%f]",
```

Comment:
Please remove commented codes

Reply:
Do we have to remove all logs? I think this log may be used later.

Reply:
Commented out code should be removed in production code when there's no special reason.

Reply:
You can create a development branch if the feature is still under development.

Reply:
Fixed it.